client/my-sites/sharing/buttons/options.jsx

```diff
@@ -211,10 +215,15 @@ const connectComponent = connect(
 		const siteId = getSelectedSiteId( state );
 		const isJetpack = isJetpackSite( state, siteId );
 		const isTwitterButtonAllowed = ! isJetpack || isJetpackMinimumVersion( state, siteId, '3.4-dev' );
+		const postTypes = values( getPostTypes( state, siteId ) );
 
 		return {
+			buttons: getSharingButtons( state, siteId ),
```

Comment:
What are your thoughts on whether we should render `<QuerySharingButtons />` and `<QuerySiteSettings />` components since we're testing against their state? Or should we assume since there's an implicit connection between this and the parent component that it's okay to rely on it already having been rendered?

(Not a loaded question, I'm on the fence personally)